collections/_posts/2022-10-17-DiffusionModel.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+author: "Sophie Carneiro Esteves"
+cite:
+    authors: "Jascha Sohl-Dickstein, Eric A. Weiss, Niru Maheswaranathan and Surya Ganguli"
+    title:   "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+    venue:   "International Conference on Machine Learning"
+pdf: "http://proceedings.mlr.press/v37/sohl-dickstein15.pdf"
+---
+
+# Notes
+- Code is available on GitHub: [https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models](https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models)
+
+# Introduction
+The authors propose a new probalistic model that aims to link a known distribution $$ \pi(.) $$ to another one $$ q(.) $$ based on diffusion. To achieve it, they iteratively added small amount of noise on the target distribution until it is converted into the known distribution. This process can be seen has a Markov chain that can be learn reversly. 
+
+
+# Highlights
+- Flexible and tractable method
+- Possibility to multiply distributions (ex : inpainting)
```

Comment:
Generally, I would expect a bit more detail here, so that we can understand the highlights somewhat independently from reading the main body of the review.

Tip: Most of the time, the highlights I use are my own interpretation of the main contributions listed by the authors in the abstract or towards the end of the introduction.